ConstraintApi.yaml

```diff
@@ -1,1156 +1,1152 @@
-        Accordingly, this URL may not have a trailing '/' character.
-      type: string
-      example: 'https://utm_uss.com/utm'
-
-    ConstraintReference:
-      description: A ConstraintReference (area in which a Constraint is present, along
-        with other high-level information, but no details).  The DSS reports only
-        these references and clients must exchange details and additional information
-        peer-to-peer.
-      required:
-        - id
-        - owner
-        - time_start
-        - time_end
-        - uss_base_url
-      type: object
-      properties:
-        id:
-          $ref: '#/components/schemas/UUIDv4'
-        owner:
-          type: string
-          example: 'uss1'
-          description: |-
-            Created by the DSS based on creating client's ID (via access token).  Used internal
-            to the DSS for restricting mutation and deletion operations to owner.  Used by USSs
-            to reject Constraint update notifications originating from a USS that does not own
-            the Constraint.
-        ovn:
-          description: |-
-            Opaque version number of this Constraint.  Populated only when the ConstraintReference
-            is owned by the USS retrieving or providing it.  Not populated when the
-            ConstraintReference is not owned by the USS retrieving or providing it (instead, the
-            USS must obtain the OVN from the details retrieved from the owning USS).
-          $ref: '#/components/schemas/EntityOVN'
-        time_start:
-          $ref: '#/components/schemas/Time'
-        time_end:
-          $ref: '#/components/schemas/Time'
-        uss_base_url:
-          $ref: '#/components/schemas/UssBaseURL'
-
-    ConstraintDetails:
-      description: Details of a UTM Constraint. Note that this data is
-        not stored in the DSS; only with the clients.
-      required:
-        - volumes
-      type: object
-      properties:
-        volumes:
-          description: |-
-            Volumes that wholly contain the Constraint while being as small as practical.
-
-            The end time may not be in the past.
-          minItems: 1
-          type: array
-          items:
-            $ref: '#/components/schemas/Volume4D'
-        type:
-          description: Type of airspace feature this Constraint represents.
-          type: string
-          example: 'NonUTMAircraftOperations'
-
-    Constraint:
-      description: Full specification of a UTM Constraint.
-      required:
-        - reference
-        - details
-      type: object
-      properties:
-        reference:
-          $ref: '#/components/schemas/ConstraintReference'
-        details:
-          $ref: '#/components/schemas/ConstraintDetails'
+---
+openapi: 3.0.1
+info:
+  title: Constraint Reporting API
+  description: 'Volume restrictions can be issued for many reasons, and this API serves as a gateway for being able to properly submit them to the appropriate FIMS'
```

Comment:
I see that we still use Volume3D/4D as our data schemas which makes sense (we're defining a volume), but is it still appropriate to call this a volume restriction here?  It feels like a reasonable synonym, but maybe just "Constraints can be issued for many reasons".

Reply:
Yeahhhhh, I did intentionally leave this out as it seemed like an unnecessary extra change, but I think any reasoning I had for it is defeated by literally anyone noticing it :D